src/style.css

```
@@ -1 +1,9 @@
 /* styles go here */
+.classForm {
+  padding-bottom: 10px;
+  display: block;
+}
```

```suggestion
.classForm {
  margin-bottom: 10px;
  display: block;
}
.classForm:last-child {
  margin-bottom: 0; 
}
```
And add this class to each label